dev/devicelab/bin/tasks/build_ios_framework_module_test.dart

```diff
@@ -939,15 +939,6 @@ Future<void> _checkStatic(String pathToLibrary) async {
   }
 }
 
-Future<String> _dylibSymbols(String pathToDylib) {
-  return eval('nm', <String>[
-    '-g',
```

Comment:
`-g` is an alias for `--extern-only`, description: `Display only global (external) symbols.`